nixos/services/ceph/client.nix

```diff
@@ -21,6 +21,30 @@ in
     flyingcircus.services.ceph = {
       config = lib.mkOption {
         type = lib.types.lines;
+        defaultText = ''
```

Comment:
Is it strictly necessary to provide a defaultText here? If not, I'd leave this one out here.

1. ceph is currently not used in our 23.11 release, the 21.05 one which is used has already diverged quite a bit.
2. The defaultText is not fully matching what's generated in the `default`, e.g `fsid` appears to be missin.
3. I see a maintenance burde of keeping the `defaultText` and the actual `default` in sync. If possible, I'd argue to generate the `defaultText` from `default` at eval time.

Reply:
Some `defaultText` is necessary since the evaluation of the `default` value implies deep evaluation of values that are not set in a simple image that only serves to generate the documentation.

If the module is not used and there is a maintenance burden I would instead suggest we provide an empty or placeholder `defaultText`.

Reply:
Yes, then let's move forward with such a placeholder.